test/package/packageVersionCreate.test.ts

```diff
@@ -263,9 +263,10 @@ describe('Package Version Create', () => {
   it('should validate options when package type = unlocked (scripts)', async () => {
     packageTypeQuery.restore();
     // @ts-ignore
-    packageTypeQuery = $$.SANDBOX.stub(connection.tooling, 'query').resolves({
-      records: [{ ContainerOptions: 'Unlocked' }],
-    });
+    packageTypeQuery = $$.SANDBOX.stub(connection.tooling, 'query')
```

Comment:
I _think_ we can remove the ts-ignores if we use `stubMethod` from ts-sinon.  Perhaps another effort as well...

Reply:
+1 on next round.